itou/api/applicants_api/tests.py

```diff
@@ -0,0 +1,119 @@
+        # Connected user must only be member of target SIAE
+        user = SiaeFactory(with_membership=True).members.first()
+        SiaeFactory(with_membership=True).members.add(user)
+
+        self.client.force_authenticate(user)
+        response = self.client.get(self.url, format="json")
+
+        self.assertEqual(response.status_code, 403)
+
+    def test_api_user_is_admin(self):
+        # Connected user must only be admin of target SIAE
+        user = SiaeFactory(with_membership=True).members.first()
+        membership = user.siaemembership_set.first()
+        membership.is_admin = False
+        membership.save()
+
+        self.client.force_authenticate(user)
+        response = self.client.get(self.url, format="json")
+
+        self.assertEqual(response.status_code, 403)
+
+    def test_login_as_siae(self):
+        # Connect with an admin user with member of a sigle SIAE
+        user = SiaeFactory(with_membership=True).members.first()
+        self.client.force_authenticate(user)
+
+        response = self.client.get(self.url, format="json")
+        self.assertEqual(response.status_code, 200)
+
+    def test_applicant_data(self):
+        siae = SiaeFactory(with_membership=True)
+        job_seeker = JobApplicationFactory(to_siae=siae).job_seeker
+        # Will not refactor ASP factories:
+        # - too long,
+        # - not the point
+        # - scheduled in a future PR
+        # => will use some hard-coded values until then
+        job_seeker.address_line_1 = "address test"
+        job_seeker.address_line_2 = "address 2"
+        job_seeker.post_code = "37000"
+        job_seeker.city = "TOURS"
+        job_seeker.resume_link = "https://myresume.com/me"
+        job_seeker.birth_place = CommuneFactory()
+        job_seeker.birth_country = CountryFactory()
+        job_seeker.save()
+        user = siae.members.first()
+
+        self.client.force_authenticate(user)
+        response = self.client.get(self.url, format="json")
+
+        self.assertEqual(response.status_code, 200)
+
+        results = response.json().get("results")
+
+        self.assertIsNotNone(results)
+        self.assertEqual(1, len(results))
+        self.assertEqual(
+            {
+                "civilite": job_seeker.title,
+                "nom": job_seeker.first_name,
+                "prenom": job_seeker.last_name,
+                "courriel": job_seeker.email,
+                "telephone": job_seeker.phone,
+                "adresse": job_seeker.address_line_1,
+                "complement_adresse": job_seeker.address_line_2,
+                "code_postal": job_seeker.post_code,
+                "ville": job_seeker.city,
+                "date_naissance": f"{job_seeker.birthdate:%d-%m-%Y}",
+                "lieu_naissance": job_seeker.birth_place.name,
+                "pays_naissance": job_seeker.birth_country.name,
+                "lien_cv": job_seeker.resume_link,
+            },
+            results[0],
+        )
```

Comment:
Revient au même et plus lisible.
```suggestion
        [result] = results
        self.assertEqual(
            {
                "civilite": job_seeker.title,
                "nom": job_seeker.first_name,
                "prenom": job_seeker.last_name,
                "courriel": job_seeker.email,
                "telephone": job_seeker.phone,
                "adresse": job_seeker.address_line_1,
                "complement_adresse": job_seeker.address_line_2,
                "code_postal": job_seeker.post_code,
                "ville": job_seeker.city,
                "date_naissance": f"{job_seeker.birthdate:%d-%m-%Y}",
                "lieu_naissance": job_seeker.birth_place.name,
                "pays_naissance": job_seeker.birth_country.name,
                "lien_cv": job_seeker.resume_link,
            },
            result,
        )
```

Reply:
Un matching qui permet d'éviter 2 lignes, miam. 
J'ai perdu les habitudes de Clojure ...